src/test/lifecycle/common.js

```diff
@@ -512,3 +542,16 @@ exports.test_and_prefix_size = async function(Bucket, Key, s3) {
     assert(actualFilter.ObjectSizeGreaterThan === expectedFilter.ObjectSizeGreaterThan, 'and prefix size filter - ObjectSizeGreaterThan');
     assert(actualFilter.ObjectSizeLessThan === expectedFilter.ObjectSizeLessThan, 'and prefix size filter - ObjectSizeLessThan');
 };
+
+exports.test_rule_duplicate_id = async function(Bucket, Key, s3) {
+    const putLifecycleParams = duplicate_id_lifecycle_configuration(Bucket, Key);
+
+    try {
+        await s3.putBucketLifecycleConfiguration(putLifecycleParams);
+        // if no error occurs, explicitly fail the test
+        assert.fail('Expected error for duplicate rule ID, but request was successful');
+    } catch (error) {
+        assert(error.code === 'InvalidArgument', 'Expected InvalidArgument: duplicate ID found in the rules');
+        console.log('Expected error received: each rule must have a unique ID, duplicate ID found');
```

Comment:
Do you need this console log printing?
In case someone will have this test fails the test, it can be added.

Reply:
Yes, we can remove it. Thanks